stats/stats.go

```diff
@@ -123,6 +129,7 @@ func (s *InHeader) IsClient() bool { return s.Client }
 
 func (s *InHeader) isRPCStats() {}
 
+
```

Comment:
nix new line

Reply:
What does nix mean in this context? I'm assuming you mean delete.